src/App.js

```diff
@@ -1,27 +1,20 @@
-import logo from './logo.svg';
 import './App.css';
-import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
-
+import { Routes, Route } from 'react-router-dom';
+import Home from './pages/Home/Home';
+import Checkout from './pages/Checkout/Checkout';
+import ProductList from './pages/ProductList/ProductList';
+import Header from './components/Header/Header';
+import Footer from './components/Footer/Footer';
 function App() {
-  const { data, isLoading } = useFeaturedBanners();
-  console.log(data, isLoading);
-
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      <Routes>
```

Comment:
To avoid nesting in the App component, it is recommended to create a component called AppRouter, and include the router logic there.